test/fixtures/analyzer_test/dead_symlink

```diff
@@ -0,0 +1 @@
+
```

Comment:
@cx-henriqueAlvelos should this file be empty?

Reply:
Yes. it is a shortcut to nowhere